### ISSUE TITLE
Quick launch of apps on mobile devices

### ISSUE THREAD

**Issue by nief12** [localsend/localsend#2052]
### Is your feature request related to a problem? Please describe.

Can shortcut switches be added to the mobile dropdown shade for quick app launching?
![Screenshot_20241117_204605_com huawei android launcher](https://github.com/user-attachments/assets/761f9a1b-c314-4a3b-aa20-40607a2c88c2)

### Describe the solution you'd like

Add quick switches

**Comment by nief12** [comment]
In fact, if it can be quickly launched and have notifications displayed during the transfer process and upon completion, it would be almost as seamless as the built-in sharing function on mobile phones

**Comment by nief12** [comment]
Of course, we might need to allow certain background running permissions, but it's also good enough to use.

**Comment by Gralent** [comment]
I also need this feature. Just need to activate it in the switch, which can save phone desktop space when using many apps

**Comment by SalicIce** [comment]
+1, I need this feature too, so I can quickly toggle sending/receiving mode.
